src/index.html

```diff
@@ -8,6 +8,54 @@
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+
+      <img
+        src="images/imac.jpeg"
+        alt="imac"
+        class="card__image"
+      >
+
+      <div class="card__info">
+        <div class="card__title">
```

Comment:
You don`t need this tag wrapper just move styles to another selector and remove div.